src/pages/UsersList/UsersList.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import * as userClient from '../../clients/user-client/user';
+import * as logger from '../../services/logger';
+
+import AuthContext from '../../store/auth-context';
+
+function UsersList() {
+    const context = React.useContext(AuthContext);
+    const isAuthorized = context.isAuthorized;
+
+    const [users, setUsers] = React.useState([]);
+
+    React.useEffect(() => {
+        if (!isAuthorized) {
+            setUsers([]);
+            return;
+        }
+
+        const pagination = {page: 1};
+        userClient.getUsers(pagination)
+            .then((userList) => {
+                setUsers(userList.items);
+            })
+            .catch((error) => logger.log(error));
+    }, [isAuthorized]);
+
+    return (
+        <>
+            <h2>Users list</h2>
+            <div>
+                <ul>
+                    {
+                        users.map(user => {
```

Comment:
here you can do it without `return`:
`users.map(user => <li key={user.id}>{user.name} {user.email}</li>)`